package.json

```diff
@@ -29,7 +29,7 @@
     "eslint": "^8.34.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-zowe-explorer": "workspace:3.0.0-next.202309141150",
```

Comment:
we only have to do this with the ZE eslint package? not the ZE API package too in dependencies?